projects/wipp-frontend-lib/src/lib/images-collection/images-collection-list/images-collection-list.component.html

```diff
@@ -72,9 +72,10 @@ <h3 class="wipp-title">Images Collections</h3>
     <td mat-cell *matCellDef="let row">{{row.imagesTotalSize | bytes}}</td>
   </ng-container>
 
+    <!-- Add center alignment -->
+  <!-- [style.text-align]="center" -->
```

Comment:
So since I upgraded to Angular 9, this was not working across the app. The element to which this class used to be applied looks good without it. I'll either remove it completely or add an appropriate css class.